src/index.ts

```diff
@@ -1,4 +1,26 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { showAPopUpScreen } from './showAPopUpScreen';
+
+const app = document.querySelector('#app') as HTMLDivElement
+
+const renderStartingPage = () => {
+    const startingPage = document.querySelector('#starting-page-template') as HTMLTemplateElement;
+    app.innerHTML = startingPage.innerHTML;
+}
+const renderInfoModal = () => {
+    const infoModal = document.querySelector('#info-modal-template') as HTMLTemplateElement;
+    app.innerHTML += infoModal.innerHTML;
+}
+const createHowToPlayButton = () => {
```

Comment:
maybe standarize/unify (ujednolicić xD) the naming convention?
"HowToPlay" button is responsible for showing "Info" modal
maybe both should be named "HowToPlay" button and modal? or both "Info"